web-components/src/components/cards/ReviewCard/ReviewCard.Photo.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { CardMedia, CardMediaProps } from '@mui/material';
+
+import Card from '../Card';
+
+const Photo: React.FC<CardMediaProps> = ({ src }) => {
+  return (
+    <Card
+      sx={{
+        mt: 9,
+        mb: 2,
+        height: [216, 293],
+      }}
+    >
```

Comment:
```suggestion
const Photo = ({ src, sx = [] }: CardMediaProps): JSX.Element => {
  return (
    <Card
      sx={{
        height: [216, 293],
        ...(Array.isArray(sx) ? sx : [sx]),
      }}
    >
```
(then pass `sx={{ mt: 9, mb: 2 }}` where you're using) - changed this from `React.FC` as it's not passed `children`

Reply:
I see what you're saying, but this is meant to be reusable only in the ReviewCard context. The margin is one of the only reasons for this component to exist, to be DRY. That's why I kept the naming `ReviewCard.Photo`

Same for `ReviewCard.ItemDisplay` above.

I am using the sx and children suggestions now though.

Reply:
ya, that makes sense and I totally appreciate wanting to keep things dry. Also realize despite all I wrote, I didn't really explain what I'm trying to suggest so apologies if all of that seemed pedantic -- thought the code in this story was great, and was more using it to try and explain an approach I think we should take moving forward to keep things reusable. It probably makes more sense to show than type, which I can do once this code is merged because I reuse some components - sorry again if this was confusing